src/test/java/org/mockitousage/verification/VerificationWithTimeoutTest.java

```diff
@@ -45,25 +45,25 @@
 
     private ScheduledExecutorService executor;
 
+    private DelayedExecution delayedExecution;
+
+
+    
+    
     @Before
     public void setUp() {
-        executor = newSingleThreadScheduledExecutor();
+        delayedExecution = new DelayedExecution();
```

Comment:
Since we need to call close() after every test to avoid delayed calls on mocks we need need a new instance before we start every test. Am I right or did I overlook something?